### ISSUE TITLE
openrc-shutdown wrong fd?

### ISSUE THREAD

**Issue by udeved** [OpenRC/openrc#152]
When shutting down with 0.27.2 using openrc-init, the shutdown actually doesn't show the services stopped, no openrc related stuff displayed. Basically no openrc statuses on shutdown.
I only get NM or acpid messages displayed on shutdown.
Is this a bug, wrong fd, or do I have overlooked a setting?

**Comment by cromerc** [comment]
You are in the wrong terminal. I get those same messages in tty7. In tty1 the openrc messages appear during shutdown.

**Comment by udeved** [comment]
On what tty is your lightdm starting @cromerc ?
My sddm starts on tty7 like gentoo.

**Comment by cromerc** [comment]
I have lightdm running on tty7 as well.
But my point is that during shutdown/reboot it doesn't switch back to tty1 which is why we don't see the openrc messages. It used to switch to tty1 at shutdown when I ran consolekit, so this is a elogind issue I believe.

**Comment by udeved** [comment]
Right, if I downgrade to sysvinit and openrc-0.26.x, I get tty output on shutdown.

**Comment by cromerc** [comment]
Under 0.27.2 with sysvinit it still switched to tty1. The 2 changes I made were switch to elogind and to openrc-init. So if the problem isn't with elogind it means that openrc-init does not switch me back to tty1 during shutdown/restart.

**Comment by cromerc** [comment]
Most likely related:
https://github.com/artix-linux/artix-linux.github.io/issues/8

**Comment by thefallenrat** [comment]
So uhhhmmm....On #openrc irc. I saw @williamh asking to @cromerc what is missing from openrc-init that makes it work, but he's disconnected from IRC before @cromerc could reply. So I'm just gonna paste here...


>ccromer: WilliamH: not sure what is missing, but looking at xdm script it calls fgconsole which returns the current console number. Under sysvinit it is returning 1, and under openrc-init it is returning 7. Not sure why

**Comment by udeved** [comment]
I totally forgot about, this issue can be closed.
The reason was the xdm script which came from gentoo, but was made for sysvinit initab.
after [removing](https://github.com/artix-linux/packages/blob/master/displaymanager-openrc/repos/extra-any/xdm.initd) the stuff, xdm behaves as expected on shutdown, and we get openrc messages shown.

**Comment by amano-kenji** [comment]
I still have to manually switch to tty1 before reboot or shutdown if sysv-utils USE flag is enabled in openrc.

**Comment by cromerc** [comment]
> I still have to manually switch to tty1 before reboot or shutdown if sysv-utils USE flag is enabled in openrc.
> 
> What is wrong with openrc-init?

If you are using "sysv-utils" you shouldn't be using openrc-init, you should be using sysvinit obviously. openrc-init should be used when sysvinit is not used.

**Comment by amano-kenji** [comment]
```
$ equery u openrc
sysv-utils : Install sysvinit compatibility scripts for halt, init, poweroff, reboot and shutdown
```

Also, sysv-utils USE flag in openrc deletes sysvinit.

**Comment by cromerc** [comment]
Sorry, was under the impression you were using sysvinit.
